src/server/active_tran_server.cpp

```diff
@@ -174,6 +174,8 @@ active_tran_server::init_page_server_hosts (const char *db_name)
       er_clear ();
     }
   exit_code = NO_ERROR;
+  cubcomm::node first_valid_connection;
+  bool connected = false;
```

Comment:
`connected` and `valid_connection_count` serve the same purpose; could make do with just one of them for the check at the end; potentially with the count which offers more "info"

Reply:
removed connected, the count had other purposes.